mvp/api/tags_api.yaml

```diff
@@ -0,0 +1,124 @@
+openapi: "3.0.0"
+servers:
+  - url: http://localhost:8080
+info:
+  version: 0.0.1
+  title: Nobis API
+tags:
+  - name: Tags controller
+
+paths:
+  /api/tags/add/{userId}:
+      put:
+        summary: Add tags by userId
+        operationId: addTags
+        parameters:
+          - in: path
+            name: userId
+            schema:
+              type: string
+            required: true
+            description: Id of the user to add tags
+        requestBody:
+          description: An array of tags to add
+          required: true
+          content:
+            application/json:
+              schema:
+                  type: array
+                  items:
+                    type: string
+        responses:
+          '200':
+            description: OK
+          '400':
+            description: Bad request. Invalid userId.
+          '404':
+            description: A user with the specified ID was not found.
+          '500':
+            description: Unexpected error.
+  
+  /api/tags/get/{userId}:
```

Comment:
Routes via `/api/users/` seem better.
Also I think it better to remove `/api/users/{userId}/tags/calculate` route. Because you can calculate a binary vector on the application side.